File: UNIDAD 1/EXAMEN_EU3_GR10/VIAJECITOS_REST_DOTNET_GR10/02. CLICON/VIAJECITOS_REST_CLICON_GR10/VIAJECITOS_REST_CLICON_GR10/Modelos/Usuario.cs
using System;

namespace VIAJECITOS_REST_CLICON_GR10.Modelos
{
    public class Usuario
    {
        public int IdUsuario { get; set; }
        public int IdCliente { get; set; }
        public string NombreUsuario { get; set; }
        public string ClaveUsuario { get; set; }
        public string EstadoUsuario { get; set; }
        public Cliente Cliente { get; set; }
    }
}

File: UNIDAD 1/00_SOA_SIN_BDD/00. SOAP_JAVA_CONUNI_GR10/02. CLIMOV/app/src/main/java/ec/edu/monster/modelo/Usuario.kt
package ec.edu.monster.modelo

import java.io.Serializable

data class Usuario(
    val idUsuario: Int,
    val idCliente: Int,
    val nombreUsuario: String,
    val claveUsuario: String,
    val estadoUsuario: String,
    val cliente: Cliente? = null
) : Serializable